docs/toolbox/accessibility_indicators/nearby_stations.md

```diff
@@ -12,24 +12,24 @@ The **PT Nearby Station Access** analysis is used to find stops and their depart
 
 ## 1. Explanation
 
-The Public Transport (PT) Nearby Stations access analysis is a suitable tool to **visualize public transport stops and their connections** that are accessible by walking/cycling from one or more starting point(s). The nearest stops, the public transport lines available there, their frequency and the journey time to reach the stop on foot and by bike are provided as a result.
+The Public Transport (PT) Nearby Stations access analysis is a suitable tool to **visualize public transport stops and their connections** that are accessible by walking/cycling from one or more starting point(s). The nearest stops, the public transport lines available there, their frequency, and the journey time to reach the stop on foot and by bike are provided as a result.
 
-**Proximity to nearby stations** is essential for various aspects of urban life and is important urban planning. The availability of public transport connections significantly improves accessibility for residents, workers, and visitors.
+**Proximity to nearby stations** is essential for various aspects of urban life and is important in urban planning. The availability of public transport connections significantly improves accessibility for residents, workers, and visitors.
 
 
 ![Nearby Stations in GOAT](/img/toolbox/accessibility_indicators/nearby_stations/nearby_stations_example.png "Nearby Stations in GOAT")
 
 :::info 
-The calculation of PT Nearby Stations is only possible in areas where the GTFS data is integrated in GOAT. As soon as you open the tool, you will see the **geofence** in which the calculation is possible. It covers Germany, Denmark, Netherlands, Belgium, Switzerland and Austria as shown in the image below.
-
+The calculation of the nearby stations is only available for areas where the transport network of the selected routing mode is integrated into GOAT. As soon as you select a routing mode, you will see the **geofence** in which the calculation is possible.
```

Comment:
Actually, since the Nearby Stations Access indicator relies primarily on the availability of PT/GTFS data, the routing mode selected by the user doesn't have an impact on the geofence. The geofence you've added below is correct, maybe just the sentence here can say something like "...is only available for areas where the public transport network is integrated into GOAT." and remove the part about selecting a routing mode.